content/pt/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Olá, Minikube!
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "Iniciar"
+    weight: 10
+    post: >
+      <p>Pronto para meter a mão na massa? Vamos criar um cluster Kubernetes simples e executar uma aplicacão exemplo.</p>
```

Comment:
There is a typo in aplicação, there is a missing ç.

Reply:
HI @devlware! Thanks for the review.